### PR TITLE
NewDetection: NewCountryValidCreds.yaml

### PR DESCRIPTION
**Pull request by lem0w**
Required items, please complete
   
   Change(s):
   - New Detection Idea

   Reason for Change(s):
   - New detection, no similar detection found on the sign-in folder.

   Version Updated:
   - Yes

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Detections/SigninLogs/NewCountryValidCreds.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  | extend CountryDiffCount = array_length(CountryDiff)
+  // Count the new column to only alert if there is a difference between both arrays
+  | where CountryDiffCount != 0
+  | extend NewCountryEvent = CountryDiff
+  | summarize by UserPrincipalName, LastObservedTime, tostring(NewCountryEvent),tostring(KnownCountry), tostring(AppName), PasswordResult, AuthSucceeded, failureReason
+entityMappings:
+  - entityType: Account
+    fieldMappings:
+      - identifier: Name
+        columnName: UserPrincipalName
```
Comment (devikamehra) [inline Detections/SigninLogs/NewCountryValidCreds.yaml:51]: Divide the UserPrincipalName into two parts Name and Domain

**Review by devikamehra — CHANGES_REQUESTED** [review]

File: Detections/SigninLogs/NewCountryValidCreds.yaml (hunk 1 above)
Comment (lem0w) [reply Detections/SigninLogs/NewCountryValidCreds.yaml:51]: sure I can do that, so identifier will be Name , column name Name and another identifier NTDomain with column name Domain once I parsed out.

Comment (lem0w) [reply Detections/SigninLogs/NewCountryValidCreds.yaml]: The changes have been applied.

**Comment by lem0w** [comment]
Any new comments on the file? Once this request is merge I can move on with the other request: https://github.com/Azure/Azure-Sentinel/pull/8016

**Comment by devikamehra** [comment]
@lem0w Why do we have two PRs for this one? I have mentioned my comments in #8016

**Comment by lem0w** [comment]
> @lem0w Why do we have two PRs for this one? I have mentioned my comments in #8016

Hi, this was the original PR for the new detection, then I realized that the anomaly sign in one was on the incorrect folder. So i'll be  closing this PR and only work on https://github.com/Azure/Azure-Sentinel/pull/8016 since that one has the folder correction and the new detection with your comments Thanks!

**Pull request closed without merging**